notebooks/403-action-recognition-webcam/403-action-recognition-webcam.ipynb

```diff
@@ -0,0 +1,646 @@
+    "import notebook_utils as utils\n",
+    "\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## The models\n",
+    "\n",
+    "### Download the models\n",
+    "\n",
+    "We use `omz_downloader`, which is a command-line tool from the `openvino-dev` package. `omz_downloader` automatically creates a directory structure and downloads the selected model.\n",
+    "\n",
+    "In this case you can use `\"action-recognition-0001\"` as a model name, and the system automatically downloads the two models `\"action-recognition-0001-encoder\"` and `\"action-recognition-0001-decoder\"`\n",
+    "\n",
+    "If you want to download another model (`\"driver-action-recognition-adas-0002-encoder\"` + `\"driver-action-recognition-adas-0002-decoder\"`, and `\"i3d-rgb-tf\"`) , please change the model name. Note: Using a model outside the list can require different pre- and post-processing. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# directory where model will be downloaded\n",
+    "base_model_dir = \"model\"\n",
+    "\n",
+    "# model name as named in Open Model Zoo\n",
+    "model_name = \"action-recognition-0001\"\n",
+    "# selected precision (FP32, FP16, FP16-INT8)\n",
+    "precision = \"FP16-INT8\"\n",
+    "\n",
+    "model_path = f\"model/intel/{model_name}/{model_name}/{precision}/{model_name}.xml\"\n",
+    "model_weights_path = f\"model/intel/{model_name}/{model_name}/{precision}/{model_name}.bin\"\n",
+    "\n",
+    "if not os.path.exists(model_path):\n",
+    "    download_command = f\"omz_downloader \" \\\n",
+    "                       f\"--name {model_name} \" \\\n",
+    "                       f\"--precision {precision} \" \\\n",
+    "                       f\"--output_dir {base_model_dir}\"\n",
+    "    ! $download_command\n",
+    "    "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Check the model path of your models\n",
+    "\n",
+    "We will be able to work with the models locally. Please check that you can see your models in your local machine after your download process."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Creating the proper path for architecture and weights model files.\n",
+    "model_path_decoder = f\"model/intel/{model_name}/{model_name}-decoder/{precision}/{model_name}-decoder.xml\"\n",
+    "model_weights_path_decoder = f\"model/intel/{model_name}/{model_name}-decoder/{precision}/{model_name}-decoder.bin\"\n",
+    "\n",
+    "model_path_encoder = f\"model/intel/{model_name}/{model_name}-encoder/{precision}/{model_name}-encoder.xml\"\n",
+    "model_weights_path_encoder = f\"model/intel/{model_name}/{model_name}-encoder/{precision}/{model_name}-encoder.bin\"\n",
+    "\n",
+    "# Testing the location of *.bin and *.xml\n",
```

Comment:
That's not really what the listdir command does.

Reply:
I have changed it by `# List of files in the encoder-model folder (*.bin and *.xml)`

Reply:
If you want to do this check that the files exist, IMO it would be nice to list all the files, not just the encoder files and not the decoder files. For example with `list(Path("model").glob("**/*.*"))` (with `Path` from `pathlib`)